docs/terms/store.md

```diff
@@ -3,19 +3,107 @@ title: Store
+import TechnicalTag from '../term_tags/_tag.mdx';
+import SetupHeader from '/docs/images/universal_map/_um_setup_header.mdx'
+import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
+import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
+
+
+<UniversalMap setup='active' connect='inactive' create='active' validate='active'/> 
+
+## Overview
+
+### Definition
 
 A Store is a connector to store and retrieve information about metadata in Great Expectations.
 
-Great Expectations supports a variety of Stores for different purposes, but the most common Stores are Expectation Stores, Validations Stores, Checkpoint Stores, and Evaluation Parameter Stores (or Metric Stores). 
+### Features and promises
+
+Great Expectations supports a variety of Stores for different purposes, but the most common Stores are Expectation Stores, Validations Stores, Checkpoint Stores, and Evaluation Parameter Stores (or Metric Stores).  Data Docs Stores can also be configured for Data Doc Sites.  Each of these Stores is tailored to a specific type of information.
+
+- [**Expectation Store:**](./expectation_store.md) a connector to store and retrieve information about collections of verifiable assertions about data.  These are Stores for Expectation Suites.
+- [**Validation Result Store:**](./validation_result_store.md) a connector to store and retrieve information about objects generated when data is Validated against an Expectation Suite.
+- [**Checkpoint Store:**](./checkpoint_store.md) a connector to store and retrieve information about means for validating data in a production deployment of Great Expectations.
+- [**Evaluation Parameter Store:**](./evaluation_parameter_store.md) a connector to store and retrieve information about parameters used during Validation of an Expectation which reference simple expressions or previously generated Metrics.
+- [**Data Docs Store:**](./data_docs_store.md) a connector to store and retrieve information pertaining to Human readable documentation generated from Great Expectations metadata detailing Expectations, Validation Results, etc.
+- [**Metric Stores:**](./metric_store.md) a connector to store and retrieve information about computed attributes of data, such as the mean of a column.  These differ from the Evaluation Parameter Store in how they are formatted.  The data in a Metrics Store is intended to be used for generating reports and analyzing trends, rather than as Evaluation Parameter values.
+
+
+### Relationship to other objects
+
+Each type of Store is designed to interact with a specific subset of information, and thus interacts with a specific subset of objects in Great Expectations.  However, all Stores can be listed and modified through your Data Context.  For further information on how a given type of Store relates to other objects, please see the corresponding Store type's technical term page:
+
+- Expectation Store technical term page: Relationship to other objects
```

Comment:
could these be links?

Reply:
Yes.  In fact, they are supposed to be.  I must've missed going back to them to provide the links after I added those pages.